addons/udes_priorities/models/priority.py

```diff
@@ -26,17 +26,36 @@ class UdesPriority(models.Model):
         default=lambda self: uuid4(),
     )
 
+    @api.multi
+    @api.depends("reference")
+    def _compute_outstanding_pickings(self):
+        Picking = self.env["stock.picking"]
+        for record in self:
+            record.outstanding_pickings = Picking.search_count(
+                [
+                    ("priority", "=", record.reference),
+                    ("state", "not in", ("cancel", "done")),
+                ]
+            )
+
     def _default_sequence(self):
         max_rec = self.search([], order="sequence desc, id desc", limit=1)
         return max_rec.sequence + 1 if max_rec else 1
 
-    sequence = fields.Integer(default=_default_sequence, help="Used to order the priorities")
+    sequence = fields.Integer(
+        default=_default_sequence, help="Used to order the priorities"
+    )
     description = fields.Text(help="A description about the purpose of the priority")
     picking_type_ids = fields.Many2many(
         "stock.picking.type",
         string="Stock Transfer Types",
         help="The stock picking types this priority is visible on. No picking types means all picking types",
     )
+    outstanding_pickings = fields.Integer(
```

Comment:
Suggestion (feel free to ignore or get a 2nd opinion): To me the field name `outstanding_pickings` isn't clear as to whether it is a recordset of pickings or a count.

Suggestion may `num_outstanding_pickings` or `outstanding_picking_count` or similar.